packages/vendure-plugin-stripe-subscription/src/stripe-subscription.plugin.ts

```diff
@@ -65,9 +64,6 @@ export interface StripeSubscriptionPluginOptions {
       ...(config.paymentOptions.paymentMethodEligibilityCheckers ?? []),
       hasStripeSubscriptionProductsPaymentChecker,
     ];
-    config.apiOptions.middleware.push(
-      createRawBodyMiddleWare('/stripe-subscription*')
-    );
```

Comment:
Ahh I missed this one. This should be there, because the request's raw body is needed to validate the incoming signature of Stripe's webhook. The e2e test don't catch this, because for testing I disabled the signature checking with 
```
StripeSubscriptionPlugin.init({
          disableWebhookSignatureChecking: true,
        }),
```
This was disabled because I couldn't find a way to mock the signature creation properly.

I will look into it.